### ISSUE TITLE
Insecure repo settings fails with newer leiningen

### ISSUE THREAD

**Issue by rbuchmann** [mpenet/tardis#1]
In a new leiningen, depending on alia leads to the following error:

```
$ lein deps 
Tried to use insecure HTTP repository without TLS.
This is almost certainly a mistake; however in rare cases where it's
intentional please see `lein help faq` for details.
```

and this in the cider stacktrace:

```
Could not find artifact com.eaio.uuid:uuid:jar:3.3 in clojars (https://repo.clojars.org/)
```

, which I narrowed down to this library. I agree with leiningen that pulling dependencies over http is a serious security risk and should be fixed as soon as possible.

**Comment by levand** [comment]
+1, this is biting us as well.

**Comment by mpenet** [comment]
I somehow didn't get any notification about that one, I ll have a look but it seems this is caused by https://github.com/mpenet/tardis/blob/master/project.clj#L6

**Comment by mpenet** [comment]
It should be fixed now and available as 1.1.1, thanks for the report.
